### ISSUE TITLE
What about rename this package to use `fastify` prefix

### ISSUE THREAD

**Issue by RafaelGSS** [fastify/point-of-view#232]
## 🚀 Feature Proposal

The package name doesn't follow the defaults: `fastify-*` or `@fastify/*`. It actually is not an issue but sounds more consistent to land this package under `fastify` prefix. What do you think?

**Comment by zekth** [comment]
I agree, we should try to use the `@fastify` prefix. This can be done in major release and adding a warning in the install script

**Comment by Eomm** [comment]
> I agree, we should try to use the @fastify prefix.

I remember that we talked about this and @jsumners exposed a lot of practical cons about it (I didn't found the original discussion)

**Comment by RafaelGSS** [comment]
Maybe the next major of this package could be under `@fastify` scope? Then could reduce the efforts.

**Comment by mcollina** [comment]
> Maybe the next major of this package could be under `@fastify` scope? Then could reduce the efforts.

You'd also need to deprecate the old ones.

**Comment by RafaelGSS** [comment]
Yes, I've researched to manage two package names but would take efforts that are not worthy in the first moment.
